projects/js-packages/charts/src/components/pie-chart/stories/index.docs.mdx

```diff
@@ -0,0 +1,406 @@
+
+<Canvas of={ PieChartStories.Default } />
+
+## Overview
+
+The Pie Chart component provides a flexible, accessible, and highly customizable solution for displaying proportional data. Built on `@visx/shape`, it supports interactive tooltips, legends, and customizable styling with gap control, corner rounding, and thickness adjustment:
+
+<Source
+	language="jsx"
+	code={ `import { PieChart } from '@automattic/charts';
+
+	<PieChart
+		data={ data }
+		withTooltips={ true }
+		showLegend={ true }
+	/>` }
+/>
+
+## Basic Usage
+
+### Simple Pie Chart
+
+The simplest pie chart requires only a `data` prop with percentage-based data:
+
+<Canvas of={ PieChartStories.Default } />
+
+<Source
+	language="jsx"
+	code={ `<PieChart
+		size={ 400 }
+		data={[
+			{ label: 'MacOS', value: 30000, percentage: 23 },
+			{ label: 'Linux', value: 22000, percentage: 17 },
+			{ label: 'Windows', value: 80000, percentage: 60 },
+		]}
+	/>` }
+/>
+
+### Required Props
+
+- **`data`**: Array of `DataPointPercentage` objects containing label, value, and percentage
+- **`size`**: Diameter of the chart in pixels (when not using responsive mode)
+
+### Optional Props
+
+- **`padding`** (default: `20`): Padding around the chart in pixels
+- **`withTooltips`** (default: `false`): Enables interactive tooltips on hover
+- **`showLegend`** (default: `false`): Shows a legend for the chart data
+- **`gapScale`** (default: `0`): Scale of gaps between segments (0-1)
+- **`cornerScale`** (default: `0`): Scale of corner rounding for segments (0-1)
+
+## Chart Styling
+
+### Pie Chart (Default)
+
+The standard pie chart displays data as complete circular segments:
+
+<Canvas of={ PieChartStories.Default } />
+
+### Enhanced Pie Chart with Styling
+
+Pie charts can include gaps and rounded corners for modern appearance:
+
+<Canvas of={ DonutStories.Doughnut } />
+
+<Source
+	language="jsx"
+	code={ `<PieChart
+		size={ 400 }
+		gapScale={ 0.03 }
+		cornerScale={ 0.03 }
+		padding={ 0 }
+		data={ data }
+	/>` }
+/>
+
+## Interactive Features
+
+### Tooltips
+
+Enable interactive tooltips that display data information on hover:
+
+<Canvas of={ PieChartStories.WithTooltips } />
+
+<Source
+	language="jsx"
+	code={ `<PieChart
+		size={ 400 }
+		data={ data }
+		withTooltips={ true }
+	/>` }
+/>
+
+### Tooltips with Pie Chart
+
+Tooltips work seamlessly with pie chart variations:
+
+<Canvas of={ DonutStories.WithTooltipsDoughnut } />
+
+## Legends
+
+### Basic Legend
+
+Add a legend to help users identify chart segments:
+
+<Canvas of={ LegendStories.Default } />
+
+<Source
+	language="jsx"
+	code={ `<PieChart
+		size={ 400 }
+		data={ data }
+		showLegend={ true }
+		legendOrientation="horizontal"
+	/>` }
+/>
+
+### Legend Positioning
+
+Control legend alignment and positioning:
+
+<Canvas of={ LegendStories.AlignmentPositioning } />
+
+<Source
```

Comment:
It's not great that the legend isn't visible, maybe we use top position or make the whole story visible

![Screenshot 2025-07-25 at 4 17 35 PM](https://github.com/user-attachments/assets/190fbc66-181a-47dd-b885-8ec869e7d50e)